npm/webpack-dev-server/src/helpers/angularHandler.ts

```diff
@@ -253,7 +254,7 @@ async function getAngularCliWebpackConfig (devServerConfig: AngularWebpackDevSer
 
   const buildOptions = getAngularBuildOptions(projectConfig.buildOptions, tsConfig)
 
-  const context = createFakeContext(projectRoot, projectConfig, logging)
+  const context = createFakeContext(projectConfig.buildOptions.workspaceRoot || projectRoot, projectConfig, logging)
```

Comment:
Could this just use the `buildOptions` output from `getAngularBuildOptions` above (same on line 279)? Slightly cleaner, perhaps